pkg/common/consumer/consumer_offset_initalizer.go

```diff
@@ -0,0 +1,43 @@
+package consumer
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Shopify/sarama"
+	"go.uber.org/zap"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"knative.dev/eventing-kafka/pkg/common/kafka/offset"
+)
+
+type consumerOffsetInitializer interface {
```

Comment:
1. this name gives me the indication that it is an "initializer" , while it's actually only checking.
2. consumer is not a consumer group. 

So maybe rename it to something like "CGOffsetsMonitor" or something like that, "Poller" , "Checker" .

Reply:
LOL, you're right. I was also doing the offset initialization inside this interface but then didn't think about the changing the name.

It is changed now.